src/lib.rs

```diff
@@ -864,36 +902,6 @@ type Commitment<G> = <<G as Group>::CE as CommitmentEngineTrait<G>>::Commitment;
 type CompressedCommitment<G> = <<<G as Group>::CE as CommitmentEngineTrait<G>>::Commitment as CommitmentTrait<G>>::CompressedCommitment;
 type CE<G> = <G as Group>::CE;
 
-/// compute digest giving a collection of Serialize object
-pub fn compute_digest<G: Group, T: Serialize>(o: &[&T]) -> G::Scalar {
-  // obtain a vector of bytes representing public parameters
-  let mut bytes = vec![];
-  o.iter()
-    .for_each(|o| bytes.extend(bincode::serialize(*o).unwrap()));
-  // convert pp_bytes into a short digest
-  let mut hasher = Sha3_256::new();
-  hasher.update(&bytes);
-  let digest = hasher.finalize();
```

Comment:
Ah, I see what the issue is, and it's actually an important one. We can and should definitely accommodate this for the Supernova pps, which is the only callsite in the code base where this is called with a slice of size > 1. But this can be done in a subsequent PR.